pyproject.toml

```diff
@@ -17,8 +17,8 @@ convokit = "^2.5.3"
 pandas = "^2.0.0"
 isort = "^5.12.0"
 nltk = "^3.8.1"
-sklearn = "^0.0.post4"
 openpyxl = "^3.1.2"
+scikit-learn = "^1.2.2"
```

Comment:
Had to use `scikit-learn` instead of `sklearn` because of this message:
```
(poetry-python-project-py3.11) michaelp@MacBook-Air-3 supreme-court-ml-predictions % poetry install
Installing dependencies from lock file

Package operations: 3 installs, 0 updates, 0 removals

  • Installing et-xmlfile (1.1.0)
  • Installing openpyxl (3.1.2)
  • Installing sklearn (0.0.post4): Failed

  ChefBuildError

  Backend subprocess exited when trying to invoke get_requires_for_build_wheel
  
  The 'sklearn' PyPI package is deprecated, use 'scikit-learn'
  rather than 'sklearn' for pip commands. 
  
  Here is how to fix this error in the main use cases:
  - use 'pip install scikit-learn' rather than 'pip install sklearn'
  - replace 'sklearn' by 'scikit-learn' in your pip requirements files
    (requirements.txt, setup.py, setup.cfg, Pipfile, etc ...)
  - if the 'sklearn' package is used by one of your dependencies,
    it would be great if you take some time to track which package uses
    'sklearn' instead of 'scikit-learn' and report it to their issue tracker
  - as a last resort, set the environment variable
    SKLEARN_ALLOW_DEPRECATED_SKLEARN_PACKAGE_INSTALL=True to avoid this error
  
  More information is available at
  https://github.com/scikit-learn/sklearn-pypi-package
  
  If the previous advice does not cover your use case, feel free to report it at
  https://github.com/scikit-learn/sklearn-pypi-package/issues/new
  

  at /opt/homebrew/Cellar/poetry/1.4.1/libexec/lib/python3.11/site-packages/poetry/installation/chef.py:152 in _prepare
      148│ 
      149│                 error = ChefBuildError("\n\n".join(message_parts))
      150│ 
      151│             if error is not None:
    → 152│                 raise error from None
      153│ 
      154│             return path
      155│ 
      156│     def _prepare_sdist(self, archive: Path, destination: Path | None = None) -> Path:

Note: This error originates from the build backend, and is likely not a problem with poetry but with sklearn (0.0.post4) not supporting PEP 517 builds. You can verify this by running 'pip wheel --use-pep517 "sklearn (==0.0.post4)"'.
```